staking/types/transaction.go

```diff
@@ -328,3 +336,42 @@ func (tx *StakingTransaction) SenderAddress() (common.Address, error) {
 func (tx *StakingTransaction) From() *atomic.Value {
 	return &tx.from
 }
+
+/*
+// AsMessage returns the transaction as a core.Message.
+//
+// AsMessage requires a signer to derive the sender.
+//
+// XXX Rename message to something less arbitrary?
+func (tx *StakingTransaction) AsMessage(s Signer) (coretypes.Message, error) {
```

Comment:
why comment out this part?

Reply:
sorry, it seems this is a leftover - the AsMessage is used in validate() which itself is used in the transaction pool, only if the transaction is an AA transaction, and staked transactions can not be AA. so I'll remove it.